keyboards/latinpad/info.json

```diff
@@ -0,0 +1,36 @@
+{
+    "keyboard_name": "latinpad",
+    "url": "https://github.com/latincompass/qmk_firmware/tree/master/keyboards/latinpad",
```

Comment:
Still no hardware address.

Reply:
Sorry, It`s my fault,This all ready ,thanks a lot Sir